Compilation fails: language selection CZ

Hi again :), I tried to compile for my native language (CZ) and the compilation returns an error similar to DE #89 #100.
It's not important to me, I'm just informing. Unless I'm doing something wrong again. DE compilation works fine.

docker run --rm -it -e "BOARD_NAME=I6X" -e "CMAKE_FLAGS=PCB=I6X HELI=NO GVARS=YES LUA_COMPILER=NO MULTIMODULE=NO PPM_UNIT=PERCENT_PREC0 DEBUG=NO PCBI6X_ELRSV2=YES TRANSLATIONS=CZ" -v f:/kompilace/opentx:/opentx vitass/opentx-fw-build

[ 38%] Building CXX object radio/src/CMakeFiles/firmware.dir/telemetry/flysky_ibus.cpp.o
/tmp/opentx/radio/src/telemetry/crossfire.cpp:42:21: error: 'ZSTR_BATT_PERCENT' was not declared in this scope
make[3]: *** [radio/src/CMakeFiles/firmware.dir/build.make:336: radio/src/CMakeFiles/firmware.dir/telemetry/crossfire.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/Makefile2:145: radio/src/CMakeFiles/firmware.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:157: radio/src/CMakeFiles/firmware.dir/rule] Error 2
make: *** [Makefile:131: firmware] Error 2

--- a/radio/src/translations/cz.h.txt
+++ b/radio/src/translations/cz.h.txt
@@ -68,11 +68,20 @@
 #define LEN_TRNCHN             "\003"
 #define TR_TRNCHN              "CH1CH2CH3CH4"
 
+#if defined(PCBI6X)
+#define LEN_AUX_SERIALMODES                 "\012"
+#if defined(CLI) || defined(DEBUG)
+#define TR_AUX_SERIALMODES                  "Debug\0      ""Telem Mirror"
+#else
+#define TR_AUX_SERIALMODES                  "OFF\0        ""Telem Mirror"
+#endif
+#else
 #define LEN_AUX_SERIALMODES         "\015"
 #if defined(CLI) || defined(DEBUG)
 #define TR_AUX_SERIALMODES          "Debug\0       ""S-Port Mirror""Telemetrie\0  ""SBUS Trenér\0 ""LUA\0         "
 #else
 #define TR_AUX_SERIALMODES          "VYP\0         ""S-Port Mirror""Telemetrie\0  ""SBUS Trenér\0 ""LUA\0         "
+#endif
 #endif
 
 #define LEN_SWTYPES            "\013"
@@ -96,8 +105,17 @@
 #define LEN_COUNTRYCODES       TR("\002", "\010")
 #define TR_COUNTRYCODES        TR("US""JP""EU", "Amerika\0""Japonsko""Evropa\0 ")
 
-#define LEN_USBMODES           TR("\006", "\010")
-#define TR_USBMODES            TR("Ask\0  ""Joyst\0""Storag""Serial", "Ask\0    ""Joystick""Storage\0""Serial\0 ")
+#if defined(PCBI6X)
+#define LEN_USBDETECTMODES             TR("\004", "\004")
+#define TR_USBDETECTMODES              TR("Auto""Once", "Auto""Once")
+#endif
+
+#define LEN_USBMODES                   TR("\006", "\010")
+#if defined(PCBI6X)
+#define TR_USBMODES                    TR("Ask\0  ""Joyst\0""Storag", "Ask\0    ""Joystick""Storage\0")
+#else
+#define TR_USBMODES                    TR("Ask\0  ""Joyst\0""Storag""Serial", "Ask\0    ""Joystick""Storage\0""Serial\0 ")
+#endif
 
 #define LEN_TARANIS_PROTOCOLS  "\004"
 #if defined(PCBI6X)
@@ -422,7 +440,10 @@
 #elif defined(PCBX9E)
   #define TR_POTS_VSRCRAW              "\310F1\0""\310F2\0""\310F3\0""\310F4\0""\311S1\0""\311S2\0""\311LS\0""\311RS\0"
   #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0""\312SE\0""\312SF\0""\312SG\0""\312SH\0""\312SI\0""\312SJ\0""\312SK\0""\312SL\0""\312SM\0""\312SN\0""\312SO\0""\312SP\0""\312SQ\0""\312SR\0"
-#elif defined(PCBXLITE) || defined(PCBI6X)
+#elif defined(PCBI6X)
+  #define TR_POTS_VSRCRAW              "\310P1\0""\310P2\0"
+  #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0"
+#elif defined(PCBXLITE)
   #define TR_POTS_VSRCRAW              "\310S1\0""\310S2\0"
   #define TR_SW_VSRCRAW                "\312SA\0""\312SB\0""\312SC\0""\312SD\0"
 #elif defined(PCBX7)
@@ -531,16 +552,20 @@
 #define BREAKSPACE             " "
 #endif
 
-#if defined(PCBTARANIS)
-  #define TR_ENTER             "[ENTER]"
+#if defined(PCBTARANIS) || defined(PCBHORUS)
+  #define TR_ENTER                     "[ENTER]"
+#elif defined(PCBI6X)
+  #define TR_ENTER                     "[ OK ]"
 #else
-  #define TR_ENTER             "[MENU]"
+  #define TR_ENTER                     "[MENU]"
 #endif
 
 #if defined(PCBHORUS)
-  #define TR_EXIT              "[RTN]"
+  #define TR_EXIT                      "[RTN]"
+#elif defined(PCBI6X)
+  #define TR_EXIT                      "[CANCEL]"
 #else
-  #define TR_EXIT              "[EXIT]"
+  #define TR_EXIT                      "[EXIT]"
 #endif
 
 #if defined(PCBTARANIS)
@@ -933,6 +958,7 @@
 #define TR_MENUSENSOR          "SENZOR"
 #define TR_SENSOR              "SENZOR"
 #define TR_COUNTRYCODE         "Kód regionu"
+#define TR_USBDETECT           "USB Detect"
 #define TR_USBMODE             "USB Mode"
 #define TR_VOICELANG           "Jazyk hlasu"
 #define TR_UNITSSYSTEM         "Jednotky"
@@ -951,9 +977,9 @@
 #define TR_RESET_TELEMETRY     "Telemetrii"
 #define TR_STATISTICS          "Statistika"
 #define TR_ABOUT_US            "O nás"
-#define TR_USB_JOYSTICK        "USB Joystick (HID)"
-#define TR_USB_MASS_STORAGE    "USB Storage (SD)"
-#define TR_USB_SERIAL          "USB Serial (Debug)"
+#define TR_USB_JOYSTICK        "USB Joystick"
+#define TR_USB_MASS_STORAGE    "USB Storage"
+#define TR_USB_SERIAL          "USB Serial"
 #define TR_SETUP_SCREENS       "Setup screens"
 #define TR_MONITOR_SCREENS     "Monitory"
 #define TR_AND_SWITCH          "AND Spínač"
@@ -1220,6 +1246,7 @@
 #define ZSTR_CURR              "Curr"
 #define ZSTR_CAPACITY          "Capa"
 #define ZSTR_VFAS              "VFAS"
+#define ZSTR_BATT_PERCENT      "Bat%"
 #define ZSTR_ASPD              "ASpd"
 #define ZSTR_GSPD              "GSpd"
 #define ZSTR_HDG               "Hdg"

--- a/radio/src/translations/de.h.txt
+++ b/radio/src/translations/de.h.txt
@@ -71,7 +71,7 @@
 #define TR_TRNCHN              "CH1CH2CH3CH4"
 
 #if defined(PCBI6X)
-#define LEN_AUX_SERIALMODES                 "\014"
+#define LEN_AUX_SERIALMODES                 "\012"
 #if defined(CLI) || defined(DEBUG)
 #define TR_AUX_SERIALMODES                  "Debug\0      ""Telem Mirror"
 #else

--- a/radio/src/translations/en.h.txt
+++ b/radio/src/translations/en.h.txt
@@ -40,8 +40,6 @@
  *  \0               -ends current string
  */
 
-
-
 // NON ZERO TERMINATED STRINGS
 #define LEN_OFFON                      "\003"
 #define TR_OFFON                       "OFF""ON\0"
@@ -71,7 +69,7 @@
 #define TR_TRNCHN                      "CH1CH2CH3CH4"
 
 #if defined(PCBI6X)
-#define LEN_AUX_SERIALMODES                 "\014"
+#define LEN_AUX_SERIALMODES                 "\012"
 #if defined(CLI) || defined(DEBUG)
 #define TR_AUX_SERIALMODES                  "Debug\0      ""Telem Mirror"
 #else

